src/auditlog/diff.py

```diff
@@ -24,8 +24,9 @@ def track_field(field):
         return False
 
     # Do not track relations to LogEntry
-    if getattr(field, 'remote_field', None) is not None and field.remote_field.model == LogEntry:
```

Comment:
why this change? the logic seems exactly the same, except `getattr()` was replace with `if hasattr` then the comparison.

Django switched their deprecation warnings to not be loud by default, [pulled from their release notes](https://docs.djangoproject.com/en/2.0/releases/1.11/#deprecating-warnings-are-no-longer-loud-by-default)

> This change allows third-party apps to support both Django 1.11 LTS and Django 1.8 LTS without having to add code to avoid deprecation warnings.

Our goal then isn't to add code to avoid deprecation warnings, it is not too long until April anyways when Django 1.8 support ends so I don't think we need to make the changes in this pr.

But let me know what your thoughts are on this and why you wanted to make the change, always happy to have people submitting prs and contributing!

Reply:
The logic is a little different, before it was possible for `field.remote_field` to exist, but not `field.remote_field.model == LogEntry`. This would cause field.rel to be referenced and sound off a deprecation warning.